### DISCUSSION TITLE
Add a toggle for displaying gridlines on land under the view menu

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21700] [Feature requests]
In RCT1, an option to toggle the display of gridlines on land was available under the view dropdown menu. Currently this does not exist in OpenRCT2, nor did it exist in RCT2. Might there be a way it could be added back in?
![image](https://github.com/OpenRCT2/OpenRCT2/assets/108596959/bdf214a5-f9c6-4a09-9862-5bfb2d89ae0d)
![image](https://github.com/OpenRCT2/OpenRCT2/assets/108596959/902f18b4-5d83-4757-b8c8-e3bfbc48e641)
![image](https://github.com/OpenRCT2/OpenRCT2/assets/108596959/d4feec0a-b9b2-4092-978d-fe6d4347e433)

**Comment by MASHzj** [comment]
That's surprising since you can assign a shortcut key to this function

**Comment by Harry-Hopkinson** [comment]
This issue thread shows it was moved to the settings window.

https://github.com/OpenRCT2/OpenRCT2/issues/1422
